src/setupProxy.js

```diff
@@ -0,0 +1,15 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+module.exports = function (app) {
+  app.use(
+    '/api/**',
+    createProxyMiddleware({
+      target: 'https://staging.cara.care',
+      ws: true,
```

Comment:
maybe it would be worth trying not setting websockets to true in `test` environment, I think it might be one of the reasons why the e2e tests might be failing 🤔